### PR TITLE
docs(faq): rewrite github hosted app section

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Change heading
- Rewrite text to include keywords I got from @TWiStErRob

## Context

Fixing discussion #23175.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/faq.md [hunk 1]
```diff
@@ -29,10 +29,12 @@ The Renovate team only fixes bugs in an older version if:
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
-## When are new Renovate OSS releases added to the hosted Mend Renovate App?
+## When is Renovate's GitHub hosted app updated?
```
Comment (HonkingGoose) [inline docs/usage/faq.md:32]: Am I allowed to use the `'` character in the heading? I suspect this `'s` thing will break something in the docs or in the tests.

Comment (TWiStErRob) [reply docs/usage/faq.md:32]: you can always flip it with "of"
```suggestion
## When is the GitHub hosted app of Renovate updated?
```

**Review by TWiStErRob — APPROVED** [review]

File: docs/usage/faq.md [hunk 2]
```diff
@@ -29,10 +29,12 @@ The Renovate team only fixes bugs in an older version if:
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
-## When are new Renovate OSS releases added to the hosted Mend Renovate App?
+## When is the GitHub hosted app of Renovate updated?
```
Comment (rarkins) [inline docs/usage/faq.md:32]:
```suggestion
## When is the Mend Renovate GitHub hosted app updated with new Renovate versions?
```

**Comment by HonkingGoose** [comment]
I think we're good to go with this  PR.

**Review by rarkins — APPROVED** [review]

**Pull request merged**